src/bip44.c

```diff
@@ -72,8 +72,20 @@ bool bip44_hasShelleyPrefix(const bip44_path_t* pathSpec)
 #undef CHECK
 }
 
+bool bip44_hasMultisigPrefix(const bip44_path_t* pathSpec)
+{
+#define CHECK(cond) if (!(cond)) return false
+	CHECK(pathSpec->length > BIP44_I_COIN_TYPE);
+	CHECK(pathSpec->path[BIP44_I_PURPOSE] == (PURPOSE_MULTISIG | HARDENED_BIP32));
+	CHECK(pathSpec->path[BIP44_I_COIN_TYPE] == (ADA_COIN_TYPE | HARDENED_BIP32));
+	return true;
+#undef CHECK
+}
+
 bool bip44_hasValidCardanoWalletPrefix(const bip44_path_t* pathSpec)
```

Comment:
Some renaming is in order. My suggestion would be something like this:
CardanoWalletPrefix -> OrdinaryWalletPrefix
MultisigPrefix -> MultisigWalletPrefix
bip44_isValidStakingKeyPath -> bip44_isOrdinaryStakingKeyPath
analogously we are likely to need bip44_isMultisigStakingKeyPath (we are not going to mix the two since tx singing mode implies one or the other)